video/paco_decoder.cpp

```diff
@@ -204,6 +208,9 @@ Graphics::PixelFormat PacoDecoder::PacoVideoTrack::getPixelFormat() const {
 }
 
 void PacoDecoder::readNextPacket() {
```

Comment:
For the following switch statement, _videoTrack is always there.
_audioTrack is initialized above. Line 85 has an 'hasAudio' flag. 
Please do check out what the right flag is for the PACo file having audio.